index.html

```diff
@@ -1,12 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>To review</title>
-</head>
-<body>
-    
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <h1 id="title">TITLE</h1>
+    <p id="description">This is our project for microverse ....</p>
```

Comment:
👍 Good job using `h1` for yout title. 
But, I think you should wrap your `h1` and `p` in `header` tag, that will improve your HTML structure and make it easier to understand. For more info check this [article ](https://www.w3schools.com/tags/tag_header.asp)
```suggestion
<header>
    <h1 id="title">TITLE</h1>
    <p id="description">This is our project for microverse ....</p>
</header>
```